test/e2e/e2e_test.go

```diff
@@ -76,13 +76,5 @@ func TestE2E(t *testing.T) {
 	// 	ginkgoconfig.GinkgoConfig.SkipString = `\[Flaky\]|\[Feature:.+\]`
 	// }
 
-	var r []ginkgo.Reporter
-	if framework.DefaultConfig.Ginkgo.ReportDirectory != "" {
-		r = append(r, reporters.NewJUnitReporter(path.Join(framework.DefaultConfig.Ginkgo.ReportDirectory,
-			fmt.Sprintf("junit_%s_%02d.xml",
-				framework.DefaultConfig.Ginkgo.ReportPrefix,
-				ginkgoconfig.GinkgoConfig.ParallelNode))))
-	}
-
-	ginkgo.RunSpecsWithDefaultAndCustomReporters(t, "cert-manager e2e suite", r)
```

Comment:
Comment: Custom ginkgo reporters have been removed for various reasons https://onsi.github.io/ginkgo/MIGRATING_TO_V2#removed-custom-reporters. We are now setting the reporter as park of the ginko invocation 

https://github.com/cert-manager/cert-manager/pull/5345/files#diff-ee0f9904e5a41e0586ddea01115fd325ef7de1dd22ae41f2cba421144292f564R63